### DISCUSSION TITLE
How to Add Custom Sections to Endpoint Pages in Scramble?

### DISCUSSION THREAD

**Discussion by AlexisVS** [dedoc/scramble#664] [Q&A] [answered]
I’m currently using Scramble to document my API, and I’d like to know if it’s possible to add custom sections to the generated documentation for endpoint pages.

For example, I want to dynamically include additional information or metadata, such as a flag for precognitive requests (as defined by Laravel Precognition), or other custom data that might be relevant for my API.

Could you provide guidance on:

    Whether it’s possible to dynamically add custom sections to endpoint pages in Scramble?
    If yes, what is the best approach to achieve this (e.g., through an extension or another mechanism)?
    If no, do you plan to support this type of customization in the future?

**Comment by romalytvynenko** [comment, marked as answer]
You can add custom description to the endpoint. Simply put it in PHPDoc.

```php
/**
 * Show item.
 * 
 * This is the content of the page you can make it as long as you want.
 */
public function show()
{
    // your route's implementation
}
```
